### PR TITLE
A Bunch of Changes

### PR DESCRIPTION
**Pull request by link07**
I made a BUNCH of changes over the last half month, from the YouTube icon (to make the You part readable), to different forms that were completely un-skinned still, to how the buttons look (I dislike the gray shader, made them just flat black).

If you don't like some of the changes, I can go through and remove them, as this does also skin a couple different things :)

### CODE REVIEW

File: dark.css [hunk 1]
```diff
@@ -176,23 +206,38 @@ body,
...
-	background-image: linear-gradient(to bottom,#474747 0,#2B2B2B 100%) !important;
-	background-image: -moz-linear-gradient(top,#474747 0,#2B2B2B 100%) !important;
-	background-image: -webkit-linear-gradient(top,#474747 0,#2B2B2B 100%) !important;
-	background-image: -o-linear-gradient(top,#474747 0,#2B2B2B 100%) !important;
-	background-image: -ms-linear-gradient(top,#474747 0,#2B2B2B 100%) !important;
-	filter: progid:DXImageTransform.Microsoft.Gradient(GradientType=0, startColorStr="#FF474747", endColorStr="#FF2B2B2B") !important;
-	box-shadow: inset 0 1px 0 rgba(31, 31, 31, .45)) !important;
-	-moz-box-shadow: inset 0 1px 0 rgba(31,31,31,.45) !important;
+	/*background-image: -webkit-linear-gradient(top,#474747 0,#2B2B2B 100%) !important;
+	background-image: linear-gradient(to bottom,#474747 0,#2B2B2B 100%) !important;*/
```
Comment (stormbreakerbg) [inline dark.css]: Can you remove the commented code please? :)

File: dark.css [hunk 2]
```diff
@@ -231,27 +278,19 @@ body .yt-uix-button[disabled]*/
...
-	background-color: #454545 !important;
-	background-image: linear-gradient(to bottom, #474747 0, #2B2B2B 100%) !important;
-	background-image: -moz-linear-gradient(bottom,#474747 0,#2B2B2B 100%) !important;
-	background-image: -webkit-linear-gradient(bottom,#474747 0,#2B2B2B 100%) !important;
-	background-image: -o-linear-gradient(bottom,#474747 0,#2B2B2B 100%) !important;
-	background-image: -ms-linear-gradient(bottom,#474747 0,#2B2B2B 100%) !important;
-	filter: progid:DXImageTransform.Microsoft.Gradient(GradientType=0, startColorStr="#FF474747", endColorStr="#FF2B2B2B") !important;
+	background-color: #0e0e0e !important;
+	/*background-image: linear-gradient(to bottom, #474747 0, #2B2B2B 100%) !important;
+	background-image: -webkit-linear-gradient(bottom,#474747 0,#2B2B2B 100%) !important;*/
```
Comment (stormbreakerbg) [inline dark.css]: Comments :)

File: dark.css [hunk 3]
```diff
@@ -290,16 +330,11 @@ a.yt-uix-button-text .yt-uix-button-content {
...
 .yt-uix-button-text:active,
 .yt-uix-button-text.yt-uix-button-toggled {
-	background-image: linear-gradient(to bottom,#474747 0,#2B2B2B 100%) !important;
-	background-image: -moz-linear-gradient(top,#474747 0,#2B2B2B 100%) !important;
-	background-image: -webkit-linear-gradient(top,#474747 0,#2B2B2B 100%) !important;
-	background-image: -o-linear-gradient(top,#474747 0,#2B2B2B 100%) !important;
-	background-image: -ms-linear-gradient(top,#474747 0,#2B2B2B 100%) !important;
-	filter: progid:DXImageTransform.Microsoft.Gradient(GradientType=0, startColorStr="#FF474747", endColorStr="#FF2B2B2B") !important;
+	/*background-image: linear-gradient(to bottom,#474747 0,#2B2B2B 100%) !important;
+	background-image: -webkit-linear-gradient(top,#474747 0,#2B2B2B 100%) !important;*/
```
Comment (stormbreakerbg) [inline dark.css]: More comments :)

**Comment by stormbreakerbg** [comment]
I like this :)

@NSDex, any objections to merging this?

**Comment by NSDex** [comment]
I've been running these changes for the past day.  The all black buttons look a bit out of place in some instances but they can be tweaked later.  Go ahead and merge.

**Comment by link07** [comment]
I'll clean up the comments in a sec.

I couldn't 100% settle on a button color, and experimented with a couple different ones before pushing, I like the one it currently is the most out of what I tried, but it still looks a bit weird.

**Comment by stormbreakerbg** [comment]
Merged! Thanks for the PR!

Does anyone want to make changes before I release it? :)

**Comment by NSDex** [comment]
I just pushed f2344a21da6eb847ad099ec1b75a26d982823162.  With that, were good to release!

**Pull request merged**